src/modal/modal.scss

```diff
@@ -28,9 +66,12 @@
     z-index: 1000;
     max-width: 50%;
     min-width: 400px;
```

Comment:
Changing this min-width to 380px makes transition from `default` screen to `max-width: 400px` smoother.

Reply:
https://user-images.githubusercontent.com/83585998/158596719-6f3ba969-1db3-4305-9f46-f028ab4a41c2.mov

vs

https://user-images.githubusercontent.com/83585998/158596736-e2941d90-1038-4899-a144-f49e05688a7b.mov

Reply:
Done